Practice1/.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+indent_size = 4
```

Comment:
The indent size should be 2 instead of 4.

Reply:
Fixed